tests/foreman/cli/test_host.py

```diff
@@ -1484,42 +1474,48 @@ def test_positive_provision_baremetal_with_uefi_secureboot():
 
 @pytest.mark.skip_if_not_set('clients', 'fake_manifest')
 @pytest.fixture(scope="module")
-def katello_host_tools_repos(module_ak, module_cv, module_lce, module_org):
+def katello_host_tools_repos():
     """Create Org, Lifecycle Environment, Content View, Activation key"""
+    org = entities.Organization().create()
+    cv = entities.ContentView(organization=org).create()
+    lce = entities.LifecycleEnvironment(organization=org).create()
+    ak = entities.ActivationKey(
+        environment=lce,
+        organization=org,
+    ).create()
     setup_org_for_a_rh_repo(
         {
             'product': PRDS['rhel'],
             'repository-set': REPOSET['rhst7'],
             'repository': REPOS['rhst7']['name'],
-            'organization-id': module_org.id,
-            'content-view-id': module_cv.id,
-            'lifecycle-environment-id': module_lce.id,
-            'activationkey-id': module_ak.id,
+            'organization-id': org.id,
+            'content-view-id': cv.id,
+            'lifecycle-environment-id': lce.id,
+            'activationkey-id': ak.id,
         }
     )
     # Create custom repository content
     setup_org_for_a_custom_repo(
         {
             'url': FAKE_6_YUM_REPO,
-            'organization-id': module_org.id,
-            'content-view-id': module_cv.id,
-            'lifecycle-environment-id': module_lce.id,
-            'activationkey-id': module_ak.id,
+            'organization-id': org.id,
+            'content-view-id': cv.id,
+            'lifecycle-environment-id': lce.id,
+            'activationkey-id': ak.id,
         }
     )
     return {
-        'ak': module_ak,
-        'cv': module_cv,
-        'lce': module_lce,
-        'org': module_org,
+        'ak': ak,
+        'cv': cv,
+        'lce': lce,
+        'org': org,
     }
 
 
 @pytest.mark.skip_if_not_set('clients')
 @pytest.fixture(scope="function")
-def katello_host_tools_client(katello_host_tools_repos):
-    client = VirtualMachine(distro=DISTRO_RHEL7)
-    client.create()
+def katello_host_tools_client(katello_host_tools_repos, rhel7_contenthost):
+    client = rhel7_contenthost
```

Comment:
why alias this?

Reply:
I was just lazy, and this was easier, but agree I should rename it.